src/webgldrawer.js

```diff
@@ -1225,6 +1245,16 @@
 
         }
 
+        /**
+         * Get the canvas center
+         * @private
+         * @param {Boolean} sketch If set to true return the center point of the sketch canvas
```

Comment:
This isn't true.

Reply:
Fixed (jsdoc comment was copied over from CanvasDrawer)